gradle/lib-versions.gradle

```diff
@@ -13,8 +13,8 @@ ext.versions = [
     commonsCollections         : '3.2.2',
     commonsText                : '1.6',
 
-    springBoot                 : '2.0.6.RELEASE',
-    springCloud                : 'Finchley.RELEASE',
+    springBoot                 : '2.1.6.RELEASE',
+    springCloud                : 'Greenwich.RELEASE',
```

Comment:
during previous migration to Spring Boot 2.0 we have used jhipster-dependencies 2.0.28 (https://mvnrepository.com/artifact/io.github.jhipster/jhipster-dependencies/2.0.28)

seems like we need to increase version to 3.0.x (https://mvnrepository.com/artifact/io.github.jhipster/jhipster-dependencies/3.0.4)

Reply:
just in order to clarify: i have made all according above comment